R/session-cookie.R

```diff
@@ -146,7 +146,7 @@ sessionCookie <- function(
 
 #' Random cookie key generator
 #'
-#' Uses a cryptographically secure pseudorandom number generator from \code{sodium::\link[sodium]{random}} to generate a 64 digit hexadecimal string.  \href{https://github.com/jeroen/sodium}{'sodium'} wraps around \href{https://download.libsodium.org/doc/}{'libsodium'}.
+#' Uses a cryptographically secure pseudorandom number generator from \code{sodium::\link[sodium]{helpers}} to generate a 64 digit hexadecimal string.  \href{https://github.com/jeroen/sodium}{'sodium'} wraps around \href{https://download.libsodium.org/doc/}{'libsodium'}.
```

Comment:
I don't know if it'll need the `()`. `[sodium::helpers()]`

```suggestion
#' Uses a cryptographically secure pseudorandom number generator from [sodium::helpers] to generate a 64 digit hexadecimal string.  \href{https://github.com/jeroen/sodium}{'sodium'} wraps around \href{https://download.libsodium.org/doc/}{'libsodium'}.
```